src/routes/safe/components/Settings/SpendingLimit/NewSpendingLimit.tsx

```diff
@@ -0,0 +1,115 @@
+import { Button, Icon, Text, Title } from '@gnosis.pm/safe-react-components'
+import { Mutator } from 'final-form'
+import React from 'react'
+import GnoForm from 'src/components/forms/GnoForm'
+import GnoButton from 'src/components/layout/Button'
+import { Amount } from 'src/routes/safe/components/Settings/SpendingLimit/Amount'
+import BeneficiarySelect from 'src/routes/safe/components/Settings/SpendingLimit/BeneficiarySelect'
+import {
+  TitleSection,
+  StyledButton,
+  FooterSection,
+  FooterWrapper,
+} from 'src/routes/safe/components/Settings/SpendingLimit/index'
+import ResetTime from 'src/routes/safe/components/Settings/SpendingLimit/ResetTime'
+import TokenSelect from 'src/routes/safe/components/Settings/SpendingLimit/TokenSelect'
+import styled from 'styled-components'
+
+const FormContainer = styled.div`
+  padding: 24px;
+  align-items: center;
+  display: grid;
+  grid-template-columns: 4fr 1fr;
+  grid-template-rows: 6fr;
+  gap: 16px 8px;
+  grid-template-areas:
+    'beneficiaryInput beneficiaryScan'
+    'tokenInput .'
+    'amountInput .'
+    'resetTimeLabel resetTimeLabel'
+    'resetTimeToggle resetTimeToggle'
+    'resetTimeOption resetTimeOption';
+`
+
+const YetAnotherButton = styled(GnoButton)`
```

Comment:
Is this just a disabled state? seems like we should support it in safe-react-components.

Reply:
No, there's another issue with the SRC Button. By setting its type to `submit` I need to "double-click" it to actually submit the form.

<kbd><img alt="double-click-to-submit" src="https://user-images.githubusercontent.com/3315606/90685573-60eab400-e240-11ea-8d4e-d625ea8db1f9.gif" width="300" /></kbd>